sensu/plugins/metric-os-api.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+#
+# Makes http calls to the different components to gather metric data
+#
+# return CRITICAL if any commands fail
+# return WARNING if no commands fail, but one or more timeout
+#
+#
+# Jose L Coello Enriquez <jlcoello@us.ibm.com>
+
+import argparse
+import re
+import sys
+import time
+import socket
+import os
+import json
+import requests
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+
+state = STATE_OK
+
+metric_data = ''
+
+DEFAULT_SCHEME = '{}'.format(socket.gethostname())
```

Comment:
if no scheme is passed to the file (what the output looks like) it gets the host name of the node and uses that as a format for the output